tests/Handler/ExceptionHandlerTest.php

```diff
@@ -82,3 +93,11 @@ public function testMethodNotAllowed()
         $this->assertEquals('GET,PUT', $response->getHeaderLine('Allow'));
     }
 }
+
+class ExceptionWithHttpStatusCode extends \Exception
```

Comment:
Normally, I'd be inclined to agree, but I realistically can't see using this class anywhere else. I'd use a mock except that PHPUnit's mocking doesn't allow stubbing methods that don't exist in the parent class.